### ISSUE TITLE
update link of pretrained model

### ISSUE THREAD

**Issue by danielkaifeng** [sfm-sr-denoising/sfm#1]
hi, very nice job on SR in frequency domain.
Could you update the pretrained model in gdrive or baidu netdisk?
Thanks!

**Comment by sfm-sr-denoising** [comment]
Thank you for your positive comment! All our pre-trained models for denoising are uploaded on this repository, but for SR the models are larger and could not be uploaded here. We still cannot upload them to GDrive or Baidu NetDisk yet, because the paper is under submission and this repository has to remain anonymous. 
We will upload these models and update the link as soon as possible. In the meantime, all our networks can be obtained again by following the re-training code and guidelines. Thank you for your patience.

**Comment by danielkaifeng** [comment]
No pro, thanks for your work!

**Comment by buildist** [comment]
Hi, I was interested in comparing your method to ESRGAN-FS but I haven't been able to run the training code, it seems like `loss` and `mseloss` here are both undefined. Is something missing? https://github.com/sfm-sr-denoising/sfm/blob/master/SR/train.py#L110

**Comment by sfm-sr-denoising** [comment]
> Hi, I was interested in comparing your method to ESRGAN-FS but I haven't been able to run the training code, it seems like `loss` and `mseloss` here are both undefined. Is something missing? https://github.com/sfm-sr-denoising/sfm/blob/master/SR/train.py#L110

Hi, two lines were accidentally deleted, adding these two lines should work:
```
loss = L1criterion(output, x_data)
mseloss = MSEcriterion(output, x_data)
```
The code is updated
